extras/audioDrivers/SDL/AudioSDL.cpp

```diff
@@ -14,16 +14,21 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with Return To The Roots. If not, see <http://www.gnu.org/licenses/>.
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
 
-#include "commonDefines.h" // IWYU pragma: keep
 #include "AudioSDL.h"
 #include "SoundSDL_Effect.h"
 #include "SoundSDL_Music.h"
-#include "driver/AudioInterface.h"
-#include "driver/IAudioDriverCallback.h"
-#include "driver/Interface.h"
+
+#include <commonDefines.h> // IWYU pragma: keep
```

Comment:
NO! This include was purposely kept at the front with the pragma. This will break things

Reply:
its still WIP - but why should it break things? the order of the includes in a header file should not be critical

Reply:
It is for the *defines file which is why the clang-format file has a regex for that and the line an IWYU pragma. It defines common aliases used throught the project (`bfs`) and includes `cstdint` to be able to use the sized integer types like built-ins.

Some time ago ALL files needed the `*defines` include, I'm reducing this step by step. The `commonDefines` is a slim version of the old include that was used there and included a lot...

Reply:
I will resolve that locally as soon as I'm finished with the files